packages/shared/src/components/icons/Privacy/outlined.svg

```diff
@@ -0,0 +1,9 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <g id="iconLeft">
+    <g id="Vector">
+      <path fill-rule="evenodd" clip-rule="evenodd" d="M12 22L16.9318 19.5207C17.6393 19.165 18.2852 18.697 18.8445 18.1346C20.2246 16.747 21 14.8649 21 12.9024V5.45624C21 5.12857 20.7915 4.83765 20.4823 4.73403L12.6325 2.10319C12.2219 1.9656 11.7781 1.9656 11.3675 2.10319L3.51773 4.73403C3.20855 4.83765 3 5.12856 3 5.45624V12.9024C3 14.8649 3.77537 16.747 5.15552 18.1346C5.71485 18.697 6.36069 19.165 7.06819 19.5207L12 22ZM19.5 5.99456L12.1581 3.53396C12.0555 3.49956 11.9445 3.49956 11.8419 3.53396L4.5 5.99456V12.9024C4.5 14.4649 5.11733 15.9634 6.21618 17.0682C6.66151 17.5159 7.17571 17.8886 7.73901 18.1717L12 20.3138L16.261 18.1717C16.8243 17.8886 17.3385 17.5159 17.7838 17.0682C18.8827 15.9634 19.5 14.4649 19.5 12.9024V5.99456Z" fill="white"/>
```

Comment:
Same on these ones..
I think you want to use currentColor here instead of hardcoded fill=white?